src/popup/fxa.js

```diff
@@ -61,3 +85,7 @@ function openSyncPreferences() {
   browser.fxa.openSyncPreferences();
   window.close();
 }
+
+async function sendTelemetry(interactionType, elapsedTime) {
```

Comment:
How come this is `async`? It doesn't have any `await`...

Reply:
Whoops, will remove.